python/rtypes/dataframe/trigger_manager.py

```diff
@@ -0,0 +1,72 @@
+
+from enums import ObjectType
+import os
+
+
+class TriggerManager(object):
+    """ TriggerManager regulates the triggers in the dataframe:
+        - giving a space for triggers to live (See other managers for an approach to this)
+        - allowing users to add triggers into the manager
+        - allowing users to delete triggers into the manager
+        - preventing triggers of the same time and action from existing
+        - allow the user to add triggers with a specified priority and have that priority take affect with any overlapping triggers
+        - allow a way to access triggers saved in the manager
+    """
+
+    def __init__(self):
+        
+        # group key (always string fullname of type) to members of the group.
+        # Ex: group key: "Customer" -- > before_insert_customer, after_delete_customer, before_read_customer
+        self.trigger_group_members = dict()
+
+    #################################################
+    ### API Methods #################################
+    #################################################
+
+    def add_trigger(self, trigger_obj):
+        # can add a single Trigger class into the trigger_manager into the dataframe
+
+        # Question: Like the type_manager's add_type method, does this method need to return a set of pairs that have been added?
+
+        self.__add_type(trigger_obj)
+
+    def add_types(self, trigger_objs):
```

Comment:
Function name. Same comment for all function names that have been copy pasted by not changed.